### ISSUE TITLE
No Connetion to HP Procurve 2620

### ISSUE THREAD

**Issue by mpikzink** [ytti/oxidized#3116]
We have an problem with the **HP Procurve  2620 series**, other series of Procurve/Aruba do not have this problem. 
It is a reproducible problem on a total of 15 devices in our infrastructure of the same model with different equipment types (POE/PortCount) and firmware versions.
If you try to connect via SSH, the connection is established successfully.
Oxidized also gets a prompt, which can be seen in the debug output, but immediately afterwards the connection is closed with the raise condition `Net::SSH::Disconnect with msg "connection closed by remote host"`.

The option `ssh_no_keepalive: true` also had no effect.
Do you have any other tips on how we can continue debugging?

```
D, [2024-04-04T22:09:19.673297 #31] DEBUG -- socket[afdc]: read 832 bytes
D, [2024-04-04T22:09:19.673382 #31] DEBUG -- socket[afdc]: received packet nr 1 type 31 len 828
D, [2024-04-04T22:09:19.678197 #31] DEBUG -- socket[afdc]: queueing packet nr 2 type 21 len 20
D, [2024-04-04T22:09:19.678250 #31] DEBUG -- socket[afdc]: sent 24 bytes
D, [2024-04-04T22:09:19.678282 #31] DEBUG -- socket[afdc]: read 16 bytes
D, [2024-04-04T22:09:19.678315 #31] DEBUG -- socket[afdc]: received packet nr 2 type 21 len 12
D, [2024-04-04T22:09:19.678490 #31] DEBUG -- net.ssh.authentication.session[b004]: beginning authentication of `<switchuser>'
D, [2024-04-04T22:09:19.678581 #31] DEBUG -- socket[afdc]: queueing packet nr 3 type 5 len 28
D, [2024-04-04T22:09:19.678605 #31] DEBUG -- socket[afdc]: sent 44 bytes
D, [2024-04-04T22:09:19.873596 #31] DEBUG -- socket[afdc]: read 44 bytes
D, [2024-04-04T22:09:19.873780 #31] DEBUG -- socket[afdc]: received packet nr 3 type 6 len 28
D, [2024-04-04T22:09:19.873894 #31] DEBUG -- net.ssh.authentication.session[b004]: trying publickey
D, [2024-04-04T22:09:19.875725 #31] DEBUG -- net.ssh.authentication.agent[b018]: connecting to ssh-agent
E, [2024-04-04T22:09:19.875773 #31] ERROR -- net.ssh.authentication.agent[b018]: could not connect to ssh-agent: Agent not configured
D, [2024-04-04T22:09:19.875863 #31] DEBUG -- net.ssh.authentication.methods.publickey[b040]: trying publickey (9f:dd:77:67:a2:1d:cc:45:14:2a:01:2a:9f:b8:c3:16) alg rsa-sha2-256
D, [2024-04-04T22:09:19.875966 #31] DEBUG -- socket[afdc]: queueing packet nr 4 type 50 len 348
D, [2024-04-04T22:09:19.876018 #31] DEBUG -- socket[afdc]: sent 364 bytes
D, [2024-04-04T22:09:19.878616 #31] DEBUG -- socket[afdc]: read 228 bytes
D, [2024-04-04T22:09:19.878801 #31] DEBUG -- socket[afdc]: received packet nr 4 type 53 len 212
I, [2024-04-04T22:09:19.878868 #31]  INFO -- net.ssh.authentication.session[b004]: We'd like to keep you up to date about:
  * Software feature updates
  * New product announcements
  * Special events
Please register your products now at:  www.hpe.com/networking/register



D, [2024-04-04T22:09:19.921762 #31] DEBUG -- socket[afdc]: read 324 bytes
D, [2024-04-04T22:09:19.921956 #31] DEBUG -- socket[afdc]: received packet nr 5 type 60 len 308
D, [2024-04-04T22:09:19.923894 #31] DEBUG -- socket[afdc]: queueing packet nr 5 type 50 len 628
D, [2024-04-04T22:09:19.923942 #31] DEBUG -- socket[afdc]: sent 644 bytes
D, [2024-04-04T22:09:19.927156 #31] DEBUG -- socket[afdc]: read 0 bytes
W, [2024-04-04T22:09:20.006175 #31]  WARN -- : default/<switchname> status no_connection, retry attempt 1
D, [2024-04-04T22:09:20.007524 #31] DEBUG -- net.ssh.transport.session[b068]: establishing connection to <switchip>:22
D, [2024-04-04T22:09:20.008756 #31] DEBUG -- net.ssh.transport.session[b068]: connection established
I, [2024-04-04T22:09:20.008803 #31]  INFO -- net.ssh.transport.server_version[b07c]: negotiating protocol version
D, [2024-04-04T22:09:20.008817 #31] DEBUG -- net.ssh.transport.server_version[b07c]: local is `SSH-2.0-Ruby/Net::SSH_7.2.1 x86_64-linux-gnu'
D, [2024-04-04T22:09:20.012456 #31] DEBUG -- net.ssh.transport.server_version[b07c]: remote is `SSH-2.0-Mocana SSH 6.3'
I, [2024-04-04T22:09:20.012543 #31]  INFO -- net.ssh.transport.algorithms[b090]: sending KEXINIT
D, [2024-04-04T22:09:20.012629 #31] DEBUG -- socket[b0a4]: queueing packet nr 0 type 20 len 1436
D, [2024-04-04T22:09:20.012673 #31] DEBUG -- socket[b0a4]: sent 1440 bytes
D, [2024-04-04T22:09:20.017950 #31] DEBUG -- socket[b0a4]: read 408 bytes
D, [2024-04-04T22:09:20.017991 #31] DEBUG -- socket[b0a4]: received packet nr 0 type 20 len 404
I, [2024-04-04T22:09:20.018009 #31]  INFO -- net.ssh.transport.algorithms[b090]: got KEXINIT from server
I, [2024-04-04T22:09:20.018044 #31]  INFO -- net.ssh.transport.algorithms[b090]: negotiating algorithms
D, [2024-04-04T22:09:20.018085 #31] DEBUG -- net.ssh.transport.algorithms[b090]: negotiated:
* kex: diffie-hellman-group14-sha1
* host_key: ssh-rsa
* encryption_server: 3des-cbc
* encryption_client: 3des-cbc
* hmac_client: hmac-sha1-96
* hmac_server: hmac-sha1-96
* compression_client: none
* compression_server: none
* language_client: 
* language_server: 
D, [2024-04-04T22:09:20.018098 #31] DEBUG -- net.ssh.transport.algorithms[b090]: exchanging keys
D, [2024-04-04T22:09:20.020187 #31] DEBUG -- socket[b0a4]: queueing packet nr 1 type 30 len 268
D, [2024-04-04T22:09:20.020213 #31] DEBUG -- socket[b0a4]: sent 272 bytes
D, [2024-04-04T22:09:20.671875 #31] DEBUG -- socket[b0a4]: read 832 bytes
D, [2024-04-04T22:09:20.671963 #31] DEBUG -- socket[b0a4]: received packet nr 1 type 31 len 828
D, [2024-04-04T22:09:20.676783 #31] DEBUG -- socket[b0a4]: queueing packet nr 2 type 21 len 20
D, [2024-04-04T22:09:20.676841 #31] DEBUG -- socket[b0a4]: sent 24 bytes
D, [2024-04-04T22:09:20.676873 #31] DEBUG -- socket[b0a4]: read 16 bytes
D, [2024-04-04T22:09:20.676904 #31] DEBUG -- socket[b0a4]: received packet nr 2 type 21 len 12
D, [2024-04-04T22:09:20.677038 #31] DEBUG -- net.ssh.authentication.session[b0b8]: beginning authentication of `<switchuser>'
D, [2024-04-04T22:09:20.677113 #31] DEBUG -- socket[b0a4]: queueing packet nr 3 type 5 len 28
D, [2024-04-04T22:09:20.677136 #31] DEBUG -- socket[b0a4]: sent 44 bytes
D, [2024-04-04T22:09:20.873621 #31] DEBUG -- socket[b0a4]: read 44 bytes
D, [2024-04-04T22:09:20.873835 #31] DEBUG -- socket[b0a4]: received packet nr 3 type 6 len 28
D, [2024-04-04T22:09:20.873951 #31] DEBUG -- net.ssh.authentication.session[b0b8]: trying publickey
D, [2024-04-04T22:09:20.878228 #31] DEBUG -- net.ssh.authentication.agent[b0cc]: connecting to ssh-agent
E, [2024-04-04T22:09:20.878308 #31] ERROR -- net.ssh.authentication.agent[b0cc]: could not connect to ssh-agent: Agent not configured
D, [2024-04-04T22:09:20.878464 #31] DEBUG -- net.ssh.authentication.methods.publickey[b0f4]: trying publickey (9f:dd:77:67:a2:1d:cc:45:14:2a:01:2a:9f:b8:c3:16) alg rsa-sha2-256
D, [2024-04-04T22:09:20.878679 #31] DEBUG -- socket[b0a4]: queueing packet nr 4 type 50 len 348
D, [2024-04-04T22:09:20.878781 #31] DEBUG -- socket[b0a4]: sent 364 bytes
D, [2024-04-04T22:09:20.881456 #31] DEBUG -- socket[b0a4]: read 228 bytes
D, [2024-04-04T22:09:20.881678 #31] DEBUG -- socket[b0a4]: received packet nr 4 type 53 len 212
I, [2024-04-04T22:09:20.881756 #31]  INFO -- net.ssh.authentication.session[b0b8]: We'd like to keep you up to date about:
  * Software feature updates
  * New product announcements
  * Special events
Please register your products now at:  www.hpe.com/networking/register



D, [2024-04-04T22:09:20.925457 #31] DEBUG -- socket[b0a4]: read 324 bytes
D, [2024-04-04T22:09:20.925687 #31] DEBUG -- socket[b0a4]: received packet nr 5 type 60 len 308
D, [2024-04-04T22:09:20.930066 #31] DEBUG -- socket[b0a4]: queueing packet nr 5 type 50 len 628
D, [2024-04-04T22:09:20.930180 #31] DEBUG -- socket[b0a4]: sent 644 bytes
D, [2024-04-04T22:09:20.933270 #31] DEBUG -- socket[b0a4]: read 0 bytes
D, [2024-04-04T22:09:20.933377 #31] DEBUG -- : <switchip> raised Net::SSH::Disconnect with msg "connection closed by remote host"
D, [2024-04-04T22:09:20.933423 #31] DEBUG -- : lib/oxidized/node.rb: Oxidized::SSH failed for <switchname>
D, [2024-04-04T22:09:20.933462 #31] DEBUG -- : lib/oxidized/job.rb: Config fetched for <switchname> at 2024-04-04 22:09:20 UTC
W, [2024-04-04T22:09:21.007905 #31]  WARN -- : default/<switchname> status no_connection, retry attempt 1
D, [2024-04-04T22:09:21.007978 #31] DEBUG -- : lib/oxidized/worker.rb: Jobs running: 0 of 2 - ended: 3 of 156
D, [2024-04-04T22:09:21.008040 #31] DEBUG -- : lib/oxidized/worker.rb: Added default/<switchname> to the job queue
D, [2024-04-04T22:09:21.008070 #31] DEBUG -- : lib/oxidized/worker.rb: Jobs running: 1 of 2 - ended: 3 of 156
D, [2024-04-04T22:09:21.008200 #31] DEBUG -- : lib/oxidized/worker.rb: 1 jobs running in parallel
D, [2024-04-04T22:09:21.008126 #31] DEBUG -- : lib/oxidized/job.rb: Starting fetching process for <switchname> at 2024-04-04 22:09:21 UTC
D, [2024-04-04T22:09:21.008454 #31] DEBUG -- : lib/oxidized/input/ssh.rb: Connecting to <switchname>
D, [2024-04-04T22:09:21.008590 #31] DEBUG -- : AUTH METHODS::["publickey"]
D, [2024-04-04T22:09:21.013302 #31] DEBUG -- net.ssh.transport.session[b108]: establishing connection to <switchip>:22
D, [2024-04-04T22:09:21.014855 #31] DEBUG -- net.ssh.transport.session[b108]: connection established
I, [2024-04-04T22:09:21.014963 #31]  INFO -- net.ssh.transport.server_version[b11c]: negotiating protocol version
D, [2024-04-04T22:09:21.015001 #31] DEBUG -- net.ssh.transport.server_version[b11c]: local is `SSH-2.0-Ruby/Net::SSH_7.2.1 x86_64-linux-gnu'
D, [2024-04-04T22:09:21.018511 #31] DEBUG -- net.ssh.transport.server_version[b11c]: remote is `SSH-2.0-Mocana SSH 6.3'
I, [2024-04-04T22:09:21.018703 #31]  INFO -- net.ssh.transport.algorithms[b130]: sending KEXINIT
D, [2024-04-04T22:09:21.018894 #31] DEBUG -- socket[b144]: queueing packet nr 0 type 20 len 1436
D, [2024-04-04T22:09:21.018996 #31] DEBUG -- socket[b144]: sent 1440 bytes
D, [2024-04-04T22:09:21.019999 #31] DEBUG -- socket[b144]: read 408 bytes
D, [2024-04-04T22:09:21.020097 #31] DEBUG -- socket[b144]: received packet nr 0 type 20 len 404
I, [2024-04-04T22:09:21.020143 #31]  INFO -- net.ssh.transport.algorithms[b130]: got KEXINIT from server
I, [2024-04-04T22:09:21.020237 #31]  INFO -- net.ssh.transport.algorithms[b130]: negotiating algorithms
D, [2024-04-04T22:09:21.020332 #31] DEBUG -- net.ssh.transport.algorithms[b130]: negotiated:
* kex: diffie-hellman-group14-sha1
* host_key: ssh-rsa
* encryption_server: 3des-cbc
* encryption_client: 3des-cbc
* hmac_client: hmac-sha1-96
* hmac_server: hmac-sha1-96
* compression_client: none
* compression_server: none
* language_client: 
* language_server: 
D, [2024-04-04T22:09:21.020361 #31] DEBUG -- net.ssh.transport.algorithms[b130]: exchanging keys
D, [2024-04-04T22:09:21.025487 #31] DEBUG -- socket[b144]: queueing packet nr 1 type 30 len 268
D, [2024-04-04T22:09:21.025548 #31] DEBUG -- socket[b144]: sent 272 bytes
D, [2024-04-04T22:09:21.690981 #31] DEBUG -- socket[b144]: read 832 bytes
D, [2024-04-04T22:09:21.691113 #31] DEBUG -- socket[b144]: received packet nr 1 type 31 len 828
D, [2024-04-04T22:09:21.702815 #31] DEBUG -- socket[b144]: queueing packet nr 2 type 21 len 20
D, [2024-04-04T22:09:21.702935 #31] DEBUG -- socket[b144]: sent 24 bytes
D, [2024-04-04T22:09:21.703005 #31] DEBUG -- socket[b144]: read 16 bytes
D, [2024-04-04T22:09:21.703080 #31] DEBUG -- socket[b144]: received packet nr 2 type 21 len 12
D, [2024-04-04T22:09:21.703374 #31] DEBUG -- net.ssh.authentication.session[b158]: beginning authentication of `<switchuser>'
D, [2024-04-04T22:09:21.703549 #31] DEBUG -- socket[b144]: queueing packet nr 3 type 5 len 28
D, [2024-04-04T22:09:21.703602 #31] DEBUG -- socket[b144]: sent 44 bytes
D, [2024-04-04T22:09:21.903966 #31] DEBUG -- socket[b144]: read 44 bytes
D, [2024-04-04T22:09:21.904209 #31] DEBUG -- socket[b144]: received packet nr 3 type 6 len 28
D, [2024-04-04T22:09:21.904321 #31] DEBUG -- net.ssh.authentication.session[b158]: trying publickey
D, [2024-04-04T22:09:21.908488 #31] DEBUG -- net.ssh.authentication.agent[b16c]: connecting to ssh-agent
E, [2024-04-04T22:09:21.908566 #31] ERROR -- net.ssh.authentication.agent[b16c]: could not connect to ssh-agent: Agent not configured
D, [2024-04-04T22:09:21.908717 #31] DEBUG -- net.ssh.authentication.methods.publickey[b194]: trying publickey (9f:dd:77:67:a2:1d:cc:45:14:2a:01:2a:9f:b8:c3:16) alg rsa-sha2-256
D, [2024-04-04T22:09:21.908938 #31] DEBUG -- socket[b144]: queueing packet nr 4 type 50 len 348
D, [2024-04-04T22:09:21.909036 #31] DEBUG -- socket[b144]: sent 364 bytes
D, [2024-04-04T22:09:21.911929 #31] DEBUG -- socket[b144]: read 228 bytes
D, [2024-04-04T22:09:21.912161 #31] DEBUG -- socket[b144]: received packet nr 4 type 53 len 212
I, [2024-04-04T22:09:21.912241 #31]  INFO -- net.ssh.authentication.session[b158]: We'd like to keep you up to date about:
  * Software feature updates
  * New product announcements
  * Special events
Please register your products now at:  www.hpe.com/networking/register



D, [2024-04-04T22:09:21.953711 #31] DEBUG -- socket[b144]: read 324 bytes
D, [2024-04-04T22:09:21.953941 #31] DEBUG -- socket[b144]: received packet nr 5 type 60 len 308
D, [2024-04-04T22:09:21.958333 #31] DEBUG -- socket[b144]: queueing packet nr 5 type 50 len 628
D, [2024-04-04T22:09:21.958448 #31] DEBUG -- socket[b144]: sent 644 bytes
D, [2024-04-04T22:09:21.961642 #31] DEBUG -- socket[b144]: read 0 bytes
D, [2024-04-04T22:09:21.961749 #31] DEBUG -- : <switchip> raised Net::SSH::Disconnect with msg "connection closed by remote host"
D, [2024-04-04T22:09:21.961794 #31] DEBUG -- : lib/oxidized/node.rb: Oxidized::SSH failed for <switchname>
D, [2024-04-04T22:09:21.961836 #31] DEBUG -- : lib/oxidized/job.rb: Config fetched for <switchname> at 2024-04-04 22:09:21 UTC
W, [2024-04-04T22:09:22.009720 #31]  WARN -- : default/<switchname> status no_connection, retry attempt 2
D, [2024-04-04T22:09:22.009799 #31] DEBUG -- : lib/oxidized/worker.rb: Jobs running: 0 of 2 - ended: 3 of 156
D, [2024-04-04T22:09:22.009857 #31] DEBUG -- : lib/oxidized/worker.rb: Added default/<switchname> to the job queue
D, [2024-04-04T22:09:22.009887 #31] DEBUG -- : lib/oxidized/worker.rb: Jobs running: 1 of 2 - ended: 3 of 156
D, [2024-04-04T22:09:22.010039 #31] DEBUG -- : lib/oxidized/worker.rb: 1 jobs running in parallel
D, [2024-04-04T22:09:22.009954 #31] DEBUG -- : lib/oxidized/job.rb: Starting fetching process for <switchname> at 2024-04-04 22:09:22 UTC
D, [2024-04-04T22:09:22.010288 #31] DEBUG -- : lib/oxidized/input/ssh.rb: Connecting to <switchname>
D, [2024-04-04T22:09:22.010705 #31] DEBUG -- : AUTH METHODS::["publickey"]
D, [2024-04-04T22:09:22.011556 #31] DEBUG -- net.ssh.transport.session[b1a8]: establishing connection to <switchip>:22
D, [2024-04-04T22:09:22.013281 #31] DEBUG -- net.ssh.transport.session[b1a8]: connection established
I, [2024-04-04T22:09:22.013387 #31]  INFO -- net.ssh.transport.server_version[b1bc]: negotiating protocol version
D, [2024-04-04T22:09:22.013425 #31] DEBUG -- net.ssh.transport.server_version[b1bc]: local is `SSH-2.0-Ruby/Net::SSH_7.2.1 x86_64-linux-gnu'
D, [2024-04-04T22:09:22.016982 #31] DEBUG -- net.ssh.transport.server_version[b1bc]: remote is `SSH-2.0-Mocana SSH 6.3'
I, [2024-04-04T22:09:22.017164 #31]  INFO -- net.ssh.transport.algorithms[b1d0]: sending KEXINIT
D, [2024-04-04T22:09:22.017343 #31] DEBUG -- socket[b1e4]: queueing packet nr 0 type 20 len 1436
D, [2024-04-04T22:09:22.017443 #31] DEBUG -- socket[b1e4]: sent 1440 bytes
D, [2024-04-04T22:09:22.018421 #31] DEBUG -- socket[b1e4]: read 408 bytes
D, [2024-04-04T22:09:22.018511 #31] DEBUG -- socket[b1e4]: received packet nr 0 type 20 len 404
I, [2024-04-04T22:09:22.018554 #31]  INFO -- net.ssh.transport.algorithms[b1d0]: got KEXINIT from server
I, [2024-04-04T22:09:22.018628 #31]  INFO -- net.ssh.transport.algorithms[b1d0]: negotiating algorithms
D, [2024-04-04T22:09:22.018708 #31] DEBUG -- net.ssh.transport.algorithms[b1d0]: negotiated:
* kex: diffie-hellman-group14-sha1
* host_key: ssh-rsa
* encryption_server: 3des-cbc
* encryption_client: 3des-cbc
* hmac_client: hmac-sha1-96
* hmac_server: hmac-sha1-96
* compression_client: none
* compression_server: none
* language_client: 
* language_server: 
D, [2024-04-04T22:09:22.018735 #31] DEBUG -- net.ssh.transport.algorithms[b1d0]: exchanging keys
D, [2024-04-04T22:09:22.023855 #31] DEBUG -- socket[b1e4]: queueing packet nr 1 type 30 len 268
D, [2024-04-04T22:09:22.023916 #31] DEBUG -- socket[b1e4]: sent 272 bytes
D, [2024-04-04T22:09:22.683048 #31] DEBUG -- socket[b1e4]: read 832 bytes
D, [2024-04-04T22:09:22.683183 #31] DEBUG -- socket[b1e4]: received packet nr 1 type 31 len 828
D, [2024-04-04T22:09:22.694502 #31] DEBUG -- socket[b1e4]: queueing packet nr 2 type 21 len 20
D, [2024-04-04T22:09:22.694616 #31] DEBUG -- socket[b1e4]: sent 24 bytes
D, [2024-04-04T22:09:22.694686 #31] DEBUG -- socket[b1e4]: read 16 bytes
D, [2024-04-04T22:09:22.694759 #31] DEBUG -- socket[b1e4]: received packet nr 2 type 21 len 12
D, [2024-04-04T22:09:22.695036 #31] DEBUG -- net.ssh.authentication.session[b1f8]: beginning authentication of `<switchuser>'
D, [2024-04-04T22:09:22.695208 #31] DEBUG -- socket[b1e4]: queueing packet nr 3 type 5 len 28
D, [2024-04-04T22:09:22.695260 #31] DEBUG -- socket[b1e4]: sent 44 bytes
D, [2024-04-04T22:09:22.893903 #31] DEBUG -- socket[b1e4]: read 44 bytes
D, [2024-04-04T22:09:22.894116 #31] DEBUG -- socket[b1e4]: received packet nr 3 type 6 len 28
D, [2024-04-04T22:09:22.894223 #31] DEBUG -- net.ssh.authentication.session[b1f8]: trying publickey
D, [2024-04-04T22:09:22.898331 #31] DEBUG -- net.ssh.authentication.agent[b20c]: connecting to ssh-agent
E, [2024-04-04T22:09:22.898405 #31] ERROR -- net.ssh.authentication.agent[b20c]: could not connect to ssh-agent: Agent not configured
D, [2024-04-04T22:09:22.898552 #31] DEBUG -- net.ssh.authentication.methods.publickey[b234]: trying publickey (9f:dd:77:67:a2:1d:cc:45:14:2a:01:2a:9f:b8:c3:16) alg rsa-sha2-256
D, [2024-04-04T22:09:22.898767 #31] DEBUG -- socket[b1e4]: queueing packet nr 4 type 50 len 348
D, [2024-04-04T22:09:22.898866 #31] DEBUG -- socket[b1e4]: sent 364 bytes
D, [2024-04-04T22:09:22.902243 #31] DEBUG -- socket[b1e4]: read 228 bytes
D, [2024-04-04T22:09:22.902464 #31] DEBUG -- socket[b1e4]: received packet nr 4 type 53 len 212
I, [2024-04-04T22:09:22.902556 #31]  INFO -- net.ssh.authentication.session[b1f8]: We'd like to keep you up to date about:
  * Software feature updates
  * New product announcements
  * Special events
Please register your products now at:  www.hpe.com/networking/register



D, [2024-04-04T22:09:22.945323 #31] DEBUG -- socket[b1e4]: read 324 bytes
D, [2024-04-04T22:09:22.945551 #31] DEBUG -- socket[b1e4]: received packet nr 5 type 60 len 308
D, [2024-04-04T22:09:22.949929 #31] DEBUG -- socket[b1e4]: queueing packet nr 5 type 50 len 628
D, [2024-04-04T22:09:22.950043 #31] DEBUG -- socket[b1e4]: sent 644 bytes
D, [2024-04-04T22:09:22.953775 #31] DEBUG -- socket[b1e4]: read 0 bytes
D, [2024-04-04T22:09:22.953877 #31] DEBUG -- : <switchip> raised Net::SSH::Disconnect with msg "connection closed by remote host"
D, [2024-04-04T22:09:22.953922 #31] DEBUG -- : lib/oxidized/node.rb: Oxidized::SSH failed for <switchname>
D, [2024-04-04T22:09:22.953961 #31] DEBUG -- : lib/oxidized/job.rb: Config fetched for <switchname> at 2024-04-04 22:09:22 UTC
W, [2024-04-04T22:09:23.011526 #31]  WARN -- : default/<switchname> status no_connection, retries exhausted, giving up
```

**Comment by robertcheramy** [comment]
Please try if it is fixed in 0.30.1

**Comment by mpikzink** [comment]
> Please try if it is fixed in 0.30.1

Unfortunately, the behavior has not changed with 0.30.1.

**Comment by robertcheramy** [comment]
> Unfortunately, the behavior has not changed with 0.30.1.

Thank you for testing that - we fixed the prompt for others procurve models in 0.30.1

I also suspect a prompt issue here.
- I can manage access to a Procurve 2620, but it will probably take a few weeks.
- You can try to debug with: https://github.com/ytti/oxidized/blob/master/docs/Troubleshooting.md#oxidized-connects-to-a-supported-device-but-no-or-partial-configuration-is-collected
- With the input debug on, oxidized should have produced a log file for the device under  ~/.config/oxidized/logs/(device-ip)-ssh. Could you poste it here after removing everything you do not feel comfortable to share?

**Comment by mpikzink** [comment]
Sure, I will post a cleaned output ASAP. I also see that after the upgrade some HP devices breaks with a regex mismatch which was running before but I haven't digged deeper yet.

**Comment by mpikzink** [comment]
For this switch type the output under ~/.config/oxidized/logs/(device-ip)-ssh is empty.

**Comment by robertcheramy** [comment]
OK, thanks. I'll have a look at it, I just need time to orginise an Access to a 2620.

**Comment by robertcheramy** [comment]
Got today access to a 2620-24, fetching the configuration worked without tuning:
```
! Image stamp:    /sw/code/build/xform(RA_15_05)
!                 Aug  9 2011 09:08:18
!                 RA.15.05.0006
!                 44
! Boot Image:     Primary
!
!     No Transceiver found on any selected port.
! Image             Size (bytes) Date     Version
! ----------------- ------------ -------- --------------------
! Primary Image    :     8829329 08/09/11 RA.15.05.0006
! Secondary Image  :     8829329 08/09/11 RA.15.05.0006
!
! Boot ROM Version : RA.15.10
! Default Boot     : Primary
!
!
!  Status and Counters - General System Information
!
!   System Name        : HP-E2620-24
!   System Contact     :
!   System Location    :
!
!   MAC Age Time (sec) : 300
!
!   Time Zone          : 0
!   Daylight Time Rule : None
!
!
!   Software revision  : RA.15.05.0006        Base MAC Addr      : 24be05-6f7860
!   ROM Version        : RA.15.10             Serial Number      : ********
!   Allow V1 Modules   : Yes
!
!
!

Running configuration:

; J9623A Configuration Editor; Created on release #RA.15.05.0006
; Ver #01:01:00

hostname "HP-E2620-24"
vlan 1
   name "DEFAULT_VLAN"
   untagged 1-28
   ip address dhcp-bootp
   exit
snmp-server community "public" unrestricted
no dhcp config-file-update
password manager
password operator
```

**Comment by mpikzink** [comment]
Thanks for this information. I chaecked the devices in our setup the oldest software version on our environment is RA.15.08.0009. So I will try to downgrade. And we auth with priv/public key to our switches so even it have something todo with one of them.

**Comment by robertcheramy** [comment]
Closing issue as no reaction from @mpikzink . If the problem persists, please open a new issue.
